docs/debug_e2etests.md

```diff
@@ -36,3 +39,63 @@ For example:
 ```bash
 kubectl get azureconfigs
 ```
+
+
+## Using Helm with the control plane cluster
+
+[The Custom Resources](https://github.com/giantswarm/apiextensions/tree/master/helm) required for the tests are installed as Helm charts, so it would be a good idea to install Helm and inspect the installed charts.
+
+There are several different Helm releases, and we need to use one that's compatible with the currently installed `tiller`.
+Let's check our `tiller` version:
+
+```bash
+kubectl get deployment -n kube-system -l name=tiller,app=helm -o jsonpath="{..image}"
+```
+
+We know the version that we need, let's install Helm:
+
+```bash
+export HELM_VERSION="v2.16.2"
+curl -L "https://get.helm.sh/helm-${HELM_VERSION}-linux-amd64.tar.gz" > ./helm.tar.gz
+tar xzvf helm.tar.gz
+chmod u+x linux-amd64/helm
+sudo mv linux-amd64/helm /usr/local/bin/
+```
+
+Now we can list the installed charts
+
+```bash
+helm list
+```
+
+Or even get the values that were used to install those charts
+
+```bash
+helm get values <release-name>
+```
+
+
+## Inspecting resources on the cloud provider
+
+The operator will create resources on the cloud provider, like the tenant clusters. Sometimes it's useful to go to the provider console and see what's going on.
+On Azure we can find the resource groups being used to hold the tenant clusters.
+
+Let's go to the Azure portal. On the resource group search box we should look for resource groups whose name starts with `ci-`.
+
+We should see a list of the resource groups for our e2e tests. We need to look for the resource group name on the CircleCI job logs to know which resource group is being used on each job.
```

Comment:
Would be nice to have an example log here to show where the name of the resource group appears. Not super important though.

Reply:
Added example